var/spack/repos/builtin/packages/py-pyarrow/package.py

```diff
@@ -34,14 +35,13 @@ class PyPyarrow(PythonPackage, CudaPackage):
     depends_on("python@3.6:", type=("build", "run"), when="@3.0.0:")
     depends_on("python@3.7:", type=("build", "run"), when="@7.0.0:")
     depends_on("py-setuptools", type="build")
-    depends_on("py-setuptools@38.6.0:", type="build", when="@7.0.0:")
+    depends_on("py-setuptools@65.5.0:", type="build", when="@7.0.0:")
```

Comment:
This should only be changed for the latest release, 7 still depends on 38.6+ right?